aiosseclient.py

```diff
@@ -29,7 +32,10 @@ async def aiosseclient(url, last_id=None, **kwargs):
                     lines = []
                     continue
 
-                yield Event.parse(''.join(lines))
+                current_event = Event.parse(''.join(lines))
+                yield current_event
+                if current_event.event == "auth_revoked":
```

Comment:
First of all thanks for the filing the issue and the patch, doesn't this feel to be too specific? Maybe it's something that a client should do or maybe not. I'll apply the patch upon you response as if I trust you given you expertise on the field. Thanks 😊